zumis/2.9.4/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:20.04
+MAINTAINER Alex Sickler (sicklera@email.chop.edu)
+
+ENV ZUMIS_VERSION 2.9.4
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+
```

Comment:
Perhaps add a comment in the docker file and in the eventual readme to ignore the `./zUMIs.sh: line 7: curl: command not found` warning, as you have explained that it has the undesired effect of trying to upgrade the repo